examples/apps/ncp/Makefile.am

```diff
@@ -85,6 +85,16 @@ ot_ncp_LDFLAGS                                                        += \
     $(NULL)
 endif # OPENTHREAD_EXAMPLES_DA15000
 
+if OPENTHREAD_EXAMPLES_DA15000
```

Comment:
Intention of this commit was only to change README.md Something must go wrong with the process. I'll draw-back this pull request.